homeassistant/components/zha/select.py

```diff
@@ -198,10 +200,19 @@ def __init__(
         **kwargs: Any,
     ) -> None:
         """Init this select entity."""
-        self._attr_options = [entry.name.replace("_", " ") for entry in self._enum]
         self._cluster_handler: ClusterHandler = cluster_handlers[0]
+        if QUIRK_METADATA in kwargs:
+            self._init_from_quirks_metadata(kwargs[QUIRK_METADATA])
+        self._attr_options = [entry.name.replace("_", " ") for entry in self._enum]
         super().__init__(unique_id, zha_device, cluster_handlers, **kwargs)
 
+    def _init_from_quirks_metadata(self, entity_metadata: EntityMetadata) -> None:
+        """Init this entity from the quirks metadata."""
+        super()._init_from_quirks_metadata(entity_metadata)
+        zcl_enum__metadata: ZCLEnumMetadata = entity_metadata.entity_metadata
+        self._attribute_name = zcl_enum__metadata.attribute_name
+        self._enum = zcl_enum__metadata.enum
```

Comment:
The `__` (double underscore) in `zcl_enum__metadata ` seems to be a typo.
```suggestion
        zcl_enum_metadata: ZCLEnumMetadata = entity_metadata.entity_metadata
        self._attribute_name = zcl_enum_metadata.attribute_name
        self._enum = zcl_enum_metadata.enum
```